### PR TITLE
SNOW-181550 Update the installation instructions for the PHP PDO Driver

### PR DESCRIPTION
**Pull request by sfc-gh-dyoshinaga**
Description
Update the installation instructions for the PHP PDO Driver

- Because the driver needs to be built, move the build instructions to the beginning of the file.
- Indicate that the driver supports Windows and macOS.
- Update the Linux instructions to indicate that they apply to macOS.
- Change the build and installation instructions to use numbered steps.

Note that this just updates the installation instructions.
I'll send a separate PR to update the usage instructions.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -8,141 +8,225 @@ PHP PDO driver for Snowflake
...
+Prerequisites
 ================================================================================
 
-PHP Versions and Extensions
-----------------------------------------------------------------------
+In order to build the Snowflake PHP PDO Driver, you must have the following software installed:
 
-PHP 7.2 is supported. The following extensions are required:
+- (Windows) Visual Studio
+-
```
Comment (sfc-gh-dyoshinaga) [inline README.rst:23]: What needs to be installed for Linux and macOS?

File: README.rst [hunk 2]
```diff
@@ -8,141 +8,225 @@ PHP PDO driver for Snowflake
...
-- php_pdo_snowflake.dll
-- cacert.pem
+   where:
 
-Copy :code:`php_pdo_snowflake.dll` to the same location as :code:`php_pdo.dll` where all PHP extensions reside (usually the :code:`ext` folder in your PHP installation).
+   - :code:`<arch>` is your CPU architecture (:code:`x64` or :code:`x86`).
+   - :code:`<build>` is the type of binary that you want to build (:code:`Release` or :code:`Debug`).
+   - :code:`<visual studio version>` is the version of Visual Studio that you are using (:code:`VS14` or :code:`VS15`).
+   - :code:`<path to PHP SDK>` is the path to the directory where the PHP SDK should be downloaded.
+     **Do not create this directory.** The script creates this directory for you when downloading the PHP SDK.
```
Comment (sfc-gh-dyoshinaga) [inline README.rst:130]: Is this correct? The script won't clone the repository if the directory already exists?

File: README.rst (hunk 1 above)
Comment (sfc-gh-kwagner) [reply README.rst:23]: For Linux you need `gcc` (version 5.2 or higher) and for macOS you need `clang` (whatever version can be installed on a user's macOS version)

File: README.rst (hunk 2 above)
Comment (sfc-gh-kwagner) [reply README.rst:130]: Yea so if that directory exists then we assume that the PHP git repo has been downloaded previously and so we just skip re-downloading the git repo to save time. Users should have the script create the folder the first time and then skip the download if it has already happened

File: README.rst (hunk 1 above)
Comment (SimbaGithub) [reply README.rst:23]: jq is needed on all platforms to run test script. Nothing special besides that, the default compiler on Linux and OSX work well.

File: README.rst (hunk 2 above)
Comment (SimbaGithub) [reply README.rst:130]: Yes, that's the behavior of git clone command.

File: README.rst [hunk 3]
```diff
@@ -8,141 +8,225 @@ PHP PDO driver for Snowflake
...
+- PHP 7.2
+- the :code:`php-pdo` extension
+- the :code:`php-json` extension
 
-If the PHP is used along with an application server, e.g., nginx, apache, which is the common use case, install them and configure the PHP handler accordingly so that the PHP file can run on the web.
-For example, Nginx enables the PHP handler by adding the following config in :code:`/etc/nginx/sites-available/default`:
+.. note::
 
-.. code-block:: text
+  The rest of the instructions assume that you have also installed the ``php-fpm`` extension.
```
Comment (sfc-gh-kwagner) [inline README.rst:33]: We should also state here that it works with the regular PHP CGI and that you don't need `php-fpm`

**Review by sfc-gh-kwagner — APPROVED** [review]
LGTM other than minor comment

File: README.rst [hunk 4]
```diff
@@ -197,117 +281,9 @@ Here is an example of fetch a row:
     $dbh = null;
     echo "OK\n";
 
-Build and Test
+Running Tests For the PHP PDO Driver
```
Comment (SimbaGithub) [inline README.rst:284]: Test on Windows is different, please see the description here:
https://github.com/snowflakedb/pdo_snowflake/pull/226

File: README.rst (hunk 1 above)
Comment (sfc-gh-dyoshinaga) [reply README.rst]: Thanks! I updated the docs.

File: README.rst (hunk 3 above)
Comment (sfc-gh-dyoshinaga) [reply README.rst]: Thanks! I added this.

File: README.rst (hunk 4 above)
Comment (sfc-gh-dyoshinaga) [reply README.rst:286]: @SimbaGithub 

Could we add the doc changes to that PR so that the doc changes and the code changes are in the same PR?

Comment (sfc-gh-dyoshinaga) [reply README.rst:284]: @SimbaGithub 

i.e. I'd rather not merge in doc changes for code changes that haven't yet been merged in.

**Comment by sfc-gh-dyoshinaga** [comment]
I'm going to merge this part in. When the PR for the tests is merged in, I'll send a separate PR to update the README.rst file.

**Pull request merged**